### PR TITLE
Adjust composer.json file to allow usage of PHP 7

### PR DESCRIPTION
**Pull request by stloyd**

### CODE REVIEW

**Comment by Ener-Getick** [comment]
Then travis should also run tests on php7.

**Comment by stloyd** [comment]
@Ener-Getick Should be good now, warning from PHPUnit will fix in next PR after this one is merged.

**Comment by stloyd** [comment]
@Ener-Getick Should be good now.

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 vendor/
+.idea/
```
Comment (Ener-Getick) [inline .gitignore]: Has this been forgotten?

File: .travis.yml [hunk 2]
```diff
@@ -1,14 +1,43 @@
...
 php:
-    - 5.3
-    - 5.4
-    - 5.5
-    - 5.6
-    - hhvm
+  - 7.0
+  - 5.6
+  - 5.3
+  - hhvm
```
Comment (Ener-Getick) [inline .travis.yml]: Can we also test on `7.1` and `nightly`?

File: .travis.yml [hunk 3]
```diff
@@ -1,14 +1,43 @@
...
+    - php: 5.6
+      env: SYMFONY_VERSION=2.7.*
+    - php: 5.6
+      env: SYMFONY_VERSION=2.8.*
+    - php: 7.0
+      env: SYMFONY_VERSION=3.0.*
+    - php: 7.0
+      env: SYMFONY_VERSION=3.1.*
+  allow_failures:
+    - php: hhvm
```
Comment (Ener-Getick) [inline .travis.yml]: Why this?

File: composer.json [hunk 4]
```diff
@@ -15,19 +15,32 @@
...
+        ]
     },
     "autoload-dev": {
-        "psr-4": { "OAuth2\\Tests\\": "tests/" }
+        "psr-4": {
+            "OAuth2\\Tests\\": "tests/"
+        }
     },
+
+    "minimum-stability": "dev",
```
Comment (Ener-Getick) [inline composer.json]: Is that necessary?

**Comment by GuilhemN** [comment]
That's great, thank you @stloyd :) (sorry for the very long delay 😅)

**Pull request merged**